.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,99 @@
+name: Bug report
+description: Create a bug report to help us improve
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for making NewPipe better by reporting a bug. :hugs:
+
+        Please fill in as much information as possible about your bug so that we don't have to play "information ping-pong" but can help you immediately.
+
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: "Checklist"
+      options:
+        - label: "I am able to reproduce the bug with the [latest version](https://github.com/TeamNewPipe/NewPipe/releases/latest)."
+          required: true
+        - label: "I made sure that there are *no existing issues*, neither [open](https://github.com/TeamNewPipe/NewPipe/issues) nor [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed), which I could contribute my information to."
+          required: true
+        - label: "I am going to take the time to to fill in all the required details. I know that the bug report will be dismissed otherwise."
```

Comment:
```suggestion
        - label: "I have taken the time to fill in all the required details. I understand that the bug report could be dismissed otherwise."
```

Reply:
> When the user reads this he/she has not filled in the data

Sure, but when they have created the issue, then the checkbox is a part of it, and it reads like something they will do in the future, not like something they should already have done before opening it.

It also breaks the consistency of the tense used with other checklist items.

For reference, youtube-dl's template also has past tense checklist sentences.